includes/class-wcs-import-admin.php

```diff
@@ -560,6 +560,8 @@ public function post_request_handler() {
 					$next_step_url_params['step']    = 2;
 					$next_step_url_params['file_id'] = $file['id'];
 
+					wp_schedule_single_event( time() + 2 * DAY_IN_SECONDS, 'importer_scheduled_cleanup', array( $file['id'] ) );
```

Comment:
Looks like this is redundant. Two events are scheduled for each import file(1 for a day later and the other one is 2 days later): https://d.pr/i/uzWH2j
The `wp_import_handle_upload()` function called [a few lines earlier](https://github.com/woocommerce/woocommerce-subscriptions-importer-exporter/blob/0689ca0c1fbeb00a105aad0633e734e579db004d/includes/class-wcs-import-admin.php#L553) schedules a similar event for a day later.

Since you mentioned that the imports take considerably longer, do you want to replace the earlier event with this one or do you want to let it be and remove this?

Reply:
Ah thanks @menaka and nice catch! I'll remove this since it is already handled by `wp_import_handle_upload()`

> Since you mentioned that the imports take considerably longer, do you want to replace the earlier event with this one or do you want to let it be and remove this?

Since `wp_import_handle_upload()` was already deleting the file after 1 day and we haven't had any issues with this, it should be fine to keep it as is.